Penteract/Assets/Scripts/PlayerController.cpp

```diff
@@ -98,6 +100,7 @@ void PlayerController::Start() {
 	if (onimaru) {
 		onimaru->Disable();
 		onimaruGun = GameplaySystems::GetGameObject(fangGunUID);
+		onimaru->GetComponent<ComponentCapsuleCollider>()->Enable(); // workaround collider doesn't activate if onimaru starts disabled
```

Comment:
Make a comprovation for nullptr here, to avoid problems 
ComponentCapsuleCollider* ccc = onimaru->GetComponent<ComponentCapsuleCollider>()
if(ccc){
ccc->Enable();
}

Reply:
Already done on a commit I did